.github/workflows/buildkit.yml

```diff
@@ -189,6 +190,35 @@ jobs:
           CACHE_TO: type=gha,scope=image${{ matrix.target-stage }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  scout:
+    runs-on: ubuntu-24.04
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
we only want to scan when merged on default branch

Reply:
Do we need to cover maintenance branches as well (e.g. `v0.16`)

Reply:
Yes as follow-up if possible. I need to check how GitHub code scanning handles multiple refs within Security tab.

Reply:
@thompson-shaun Seems affected branches are correctly reported in security tab: 

![image](https://github.com/user-attachments/assets/8df706b3-f7a0-480d-beb8-005cd5f349b3)

So I guess we could scan for release branches as follow-up.